packages/mcp-server/src/tools/index.ts

```diff
@@ -0,0 +1,283 @@
+// File generated from our OpenAPI spec by Stainless. See CONTRIBUTING.md for details.
+
+import Metronome from '@metronome/sdk';
+import { Tool } from '@modelcontextprotocol/sdk/types.js';
+
+import retrieve_v2_contracts from './v2/contracts/retrieve-v2-contracts';
+import list_v2_contracts from './v2/contracts/list-v2-contracts';
+import edit_v2_contracts from './v2/contracts/edit-v2-contracts';
+import edit_commit_v2_contracts from './v2/contracts/edit-commit-v2-contracts';
+import edit_credit_v2_contracts from './v2/contracts/edit-credit-v2-contracts';
+import get_edit_history_v2_contracts from './v2/contracts/get-edit-history-v2-contracts';
+import create_v1_alerts from './v1/alerts/create-v1-alerts';
+import archive_v1_alerts from './v1/alerts/archive-v1-alerts';
+import list_v1_plans from './v1/plans/list-v1-plans';
+import get_details_v1_plans from './v1/plans/get-details-v1-plans';
+import list_charges_v1_plans from './v1/plans/list-charges-v1-plans';
+import list_v1_plans_customers from './v1/plans/list-customers-v1-plans';
```

Comment:
FYI: originally had a merge conflict with `import list_customers_v1_plans from './v1/customers/plans/list-customers-v1-plans';`, so I renamed this to `list_v1_plans_customers`